### PR TITLE
add "add a comment" to lib-to-slack pushes

### PR DESCRIPTION
**Pull request by camhashemi**
@ryanpbrewster

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/slack/LibraryToSlackChannelPusher.scala [hunk 1]
```diff
@@ -134,6 +134,10 @@ class LibraryToSlackChannelPusherImpl @Inject() (
       else basicUser.firstName --> LinkElement(pathCommander.userPageViaSlack(basicUser, slackTeamId))
     }
     val keepLink = LinkElement(if (shouldSmartRoute) pathCommander.keepPageOnUrlViaSlack(keep, slackTeamId).absolute else keep.url)
+    val addCommentOpt = {
+      if (shouldSmartRoute) Some(s"\n${SlackEmoji.speechBalloon.value} Add a comment" --> LinkElement(pathCommander.keepPageOnKifiViaSlack(keep, slackTeamId)))
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/slack/LibraryToSlackChannelPusher.scala:138]: Any way we can make this us only initially to test?

Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/slack/LibraryToSlackChannelPusher.scala:138]: `shouldSmartRoute` is doing that

Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/slack/LibraryToSlackChannelPusher.scala:138]: Sweet :+1:

File: kifi-backend/modules/shoebox/app/com/keepit/slack/LibraryToSlackChannelPusher.scala [hunk 2]
```diff
@@ -141,14 +145,14 @@ class LibraryToSlackChannelPusherImpl @Inject() (
         DescriptionElements(
           keep.title.getOrElse(keep.url.abbreviate(KEEP_URL_MAX_DISPLAY_LENGTH)) --> keepLink,
           "from", s"#${post.channel.name.value}" --> LinkElement(post.permalink),
-          "was added to", lib.name --> libLink, "."
+          "was added to", lib.name --> libLink, addCommentOpt.getOrElse(".")
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/slack/LibraryToSlackChannelPusher.scala:148]: I usually find it simpler to just put the optional `DescriptionElements` in there and keep the period separate.

So

```
"was added to", lib.name --> libLink, addCommentOpt, "."
```

Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/slack/LibraryToSlackChannelPusher.scala:148]: this is a personal design decision (by me). I like symmetry and don't like "Add a comment." with a period (it's not a sentence).

Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/slack/LibraryToSlackChannelPusher.scala:148]: so we omit the period on both lines if "Add a comment" is there

**Pull request merged**